src/util/fetchUtils.js

```diff
@@ -87,8 +87,8 @@ export const getFetchArgs = (apiUrl, queries, meta) => {
 	const isFormData = queries[0].params instanceof FormData;
 	const isDelete = method === 'DELETE';
 
-	const fetchUrl = new URL(apiUrl);
-	fetchUrl.searchParams.append('queries', rison.encode_array(queries));
+	const searchParams = new URLSearchParams();
```

Comment:
it looks like URLSearchParams gets polyfilled when the URL polyfill is supplied - https://github.com/inexorabletash/polyfill#url-api